fastlane/Fastfile

```diff
@@ -911,8 +958,38 @@ platform :ios do
   end
 
   desc 'Run localization only'
-  lane :update_localization do
-    ios_localize_project
+  lane :generate_strings_file_for_glotpress do
+    cocoapods
+
+    en_lproj_path = File.join(RESOURCES_FOLDER, 'en.lproj')
+    ios_generate_strings_file_from_code(
+      paths: [
+        'WooCommerce/',
+        'Pods/WordPress*/',
+        'Storage/Storage/',
+        'Networking/Networking/',
+        'Hardware/Hardware/',
+      ],
+      exclude: [
+        '*Vendor*',
+        'WooCommerce/WooCommerceTests/**', # Some unit tests include localized strings, but we don't want to crowd GlotPress with them
+      ],
+      output_dir: en_lproj_path
+    )
+
+    # Merge various manually-maintained `.strings` files into the previously generated `Localizable.strings` so their extra keys are also imported in GlotPress.
+    #
+    # Note: We will re-extract the translations back during `download_localized_strings_from_glotpress` (via a call to `ios_extract_keys_from_strings_files`)
+    ios_merge_strings_files(
+      paths_to_merge: MANUALLY_MAINTAINED_STRINGS_FILES,
+      destination: File.join(en_lproj_path, 'Localizable.strings')
+    )
+
+    git_commit(
+      path: en_lproj_path,
+      message: 'Update strings for localization',
```

Comment:
Super-nit: I wonder if we shouldn't start using a term like "freeze" / "string freeze" as a parallel to "code freeze" (here and ideally ultimately in the other repos too, of course, including Android which has slightly different implementation details for that step but does the correpsonding steps with the same intention as here).

After all, what we're doing here is "freezing" the strings that are in code, the same way we freeze the code itself (except that given GlotPress imports files that are in `trunk` and not in `release/*` branches, we need to only update those `.strings` files during "code (& strings) freeze").

That might be a bit more explicit in the Android side, because for Android what we do is that we just _copy_ the `src/main/res/values/strings.xml` file into `fastlane/resources/values/strings.xml` (which is the file that gets imported by GlotPress), so it might be easier/more natural to see that copy of the file as a "freeze" of said file. While here for iOS we don't "make a copy of a file to freeze it", but instead "only generate the strings file from code during code freeze". But still, the idea is the same, and `"Freeze strings for translation"` might thus be a more explicit commit message?

wdyt?

Reply:
I like the symmetry 👍 The `.strings` file is not really frozen, but neither is the code, really 😅 Or, let's say they're frozen in the same way ❄️

Reply:
```suggestion
      message: 'Freeze strings for localization',
```